.gitignore

```diff
@@ -48,6 +48,7 @@ api-spec.json
 # Edit at https://www.toptal.com/developers/gitignore?templates=visualstudiocode
 
 ### VisualStudioCode ###
+.vscode/
```

Comment:
is this needed?

Reply:
probably not, likely left there from before i added the official one. removed!